### PR TITLE
Moving translations section

### PR DESCRIPTION
**Pull request by esthermmoturi**
# Description

Closes https://github.com/medic/cht-docs/issues/1640

### CODE REVIEW

**Review by andrablaj — COMMENTED** [review]
All links/images/aliases look good. 👌 

I have some questions about the hierarchy:

- The `Overview` page contains more details than just an overview, such as `Implementation Steps`. What do you think about splitting the Implementation Steps to a page called `Implementing`?
- The `Configuring` page seems to be more about managing translations, than configuring them. I suggest renaming it to `Managing` or `Adding (Translations)`.

**Comment by esthermmoturi** [comment]
Yes, the Overview page is quite detailed. I can split the pages then rename the one with implementation steps as Configuring, what do you think?

File: content/en/building/translations/managing-translations.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
-title: "How To Manage Translations"
-linkTitle: "Translations"
-weight: 5
+title: Managing
+linkTitle: Managing
+weight: 2
 description: >
   Process for managing translations in CHT Core
```
Comment (andrablaj) [inline content/en/building/translations/managing-translations.md:6]:
```suggestion
  Adding translations in CHT Core
```

**Review by andrablaj — APPROVED** [review]
Looks good! Added two non-blocking suggestions.

**Pull request merged**